main.py

```diff
@@ -106,6 +109,8 @@ def update_score(self):
             STAR -= 0.5
         elif self.type == ItemType.BONUS:
             SCORE += 5
+        elif self.type == ItemType.SLOWDOWN:
+            STAR -= 1
```

Comment:
You are decresing star when player collide with slow down item, not player.speed